File: max-element/boilerplate-full/function.js

	##USER_CODE_HERE##
		
  const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
  const size = Number(input.shift());
const arr = input.shift().split(' ').map(Number);
  const result = maxElement(size, arr);
  console.log(JSON.stringify(result));
      

File: max-element/boilerplate-full/function.java

  import java.util.*;
  
  class Main {
      public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int size = scanner.nextInt();
scanner.nextLine(); // consume newline
        int[] arr = Arrays.stream(scanner.nextLine().trim().split("\\s+"))
                    .mapToInt(Integer::parseInt)
                    .toArray();
		Main m = new Main(); 
		Solution solution = m.new Solution();
		int result = solution.maxElement(size, arr);
		System.out.println(result);
		scanner.close();
      }

	  ##USER_CODE_HERE##
  }
      